tests/test_utils.py

```diff
@@ -887,8 +887,91 @@ def test_bucket_is_valid_invalid_suffix():
 
 
 def test_bucket_is_valid_ok():
-    """Test that a bucket name with suffix -s3alias is not valid."""
+    """Test that a bucket name with suffix -s3alias is valid."""
```

Comment:
```suggestion
    """Test that a bucket name with suffix -s3alias is not valid."""
```

Reply:
I don't get this one, the previous test seem to deal with not valid name and this one asserts valid. It should probably be something like """Test that a bucket name without suffix is valid."""